src/main/java/UseCases/PresenterDisplay.java

```diff
@@ -1,55 +1,46 @@
 package UseCases;
+import Entities.State;
 import Interactors.InputInteractor;
 import Interactors.OutputInteractor;
 import Persistence.LoadFile;
 
+import GUI.JDisplay;
 import java.io.File;
-import java.util.Scanner;
 
 /**
  * UseCases.PresenterDisplay is a class that runs the game loop and presents the options of each turn to each
  * player in the game.
  **/
 public class PresenterDisplay {
-    /**
-     * InstanceVar isOver: variable to check if the game is over
-     **/
-    private static boolean isOver;
 
     /**
-     * This is the constructor for the UseCases.PresenterDisplay Class
-     **/
-    public PresenterDisplay(){
-        isOver = false;
-    }
-
-
-    /**
-     * Function that runs the game loop by getting game data from the IOController and presenting that to the
-     * user as their options on each turn, getting the user's input and sending that back to the IOController
+     * Function that runs the game loop by getting game data from the OutputInteractor and presenting that to the
+     * user as their options on each turn through Conversion in the JDisplay,
+     *  and then getting the user's input and sending that back to the InputInteractor
      * to further handle state changes based on their option choice.
      **/
     public void playGame(File file){
         UseCaseInteractor interactor = new UseCaseInteractor(new LoadFile(file));
```

Comment:
I believe UseCaseInteractor takes in 2 parameters now. There are a few changes made in UseCaseInteractor class.